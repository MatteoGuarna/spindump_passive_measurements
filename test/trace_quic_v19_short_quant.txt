Simple connection with the Quant implementation using the EGG-19 (v19) version number.
