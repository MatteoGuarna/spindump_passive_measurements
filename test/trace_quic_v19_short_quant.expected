QUIC 172.20.10.6 <-> 52.58.13.57 f4f51c2a-35bd7f1b1ceb29d62e3227ab6c6b (49623:4433) at 1554975101813067 new starting packets 0 0 bytes 0 0
QUIC 172.20.10.6 <-> 52.58.13.57 f4f51c2a-a150f23a0d078a69 (49623:4433) at 1554975101966715 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0
QUIC 172.20.10.6 <-> 52.58.13.57 f4f51c2a-a150f23a0d078a69 (49623:4433) at 1554975101966715 measurement up right 153648 packets 1 0 bytes 1228 0 bandwidth 1228 0
